pkg/environmet/env.go

```diff
@@ -30,6 +30,13 @@ const (
 
 	// SkipVerify the client will skip tls verification if this is true
 	SkipVerify = "TYK_TLS_INSECURE_SKIP_VERIFY"
+	// IngressClasses is comma separated list of ingress class name that the
+	// operator will listen to. These will added to the default value tyk.
+	//
+	// Example if you set this WATCH_INGRESS_CLASSES="nginx,custom". Then the
+	// operator will start checking tyk,nginx and custom, class name, any event
+	// that has one of the class will reconciled
+	IngressClasses = "WATCH_INGRESS_CLASSES"
```

Comment:
The ingress controller should only change 1 ingress class.

Reply:
fixed at 09ac635